Sources/FeaturevisorSDK/Instance.swift

```diff
@@ -370,7 +428,9 @@ public class FeaturevisorInstance {
                     enabled: stickyFeature.enabled,
                     sticky: stickyFeature)
 
-                  logger.debug("using sticky enabled", ["featureKey": featureKey]) // TODO: Log evaluation object. Make it encodable
+            let evaluationDictionary = toDictionary(evaluation)
+
+            logger.debug("using sticky enabled", ["featureKey": evaluationDictionary])
```

Comment:
You still assign everything to featureKey. As I mentioned earlier, it should be just like
`logger.debug("using sticky enabled",  evaluationDictionary`